PSKoans/ExportedTypes.ps1

```diff
@@ -11,4 +28,58 @@ class KoanAttribute : System.Attribute {
     KoanAttribute() {
         $this.Position = [uint32]::MaxValue
     }
+
+    [Object] InvokePester([HashTable]$PesterParams) {
+        Write-Verbose "Testing karma with file [$($this.FileCmdlet.Name)]"
+
+        $PesterParams.Script = $this.FilePath
+        $this.FilePesterTests = Invoke-Pester @PesterParams
+        return $this.FilePesterTests
+    }
+
+    [int] GetFailedCount() {
+        return $this.FilePesterTests.FailedCount
+    }
+
+    [PSObject] GetTestResults([string]$Type) {
+        switch ($Type) {
+            'Failed' {
+                return $this.FilePesterTests.TestResult | Where-Object Result -eq $Type
+            }
+            Default: {
+                throw "Not Expected"
```

Comment:
Wouldn't it just be better to have the method named `GetFailedTests()` and skip the need for the switch, if it's not going to be used?